docs/contributing/testing.md

```diff
@@ -120,7 +120,25 @@ If a snapshot test fails, follow these steps.
 
 Where `<COMPONENT>` is the name of the component you've changed.
 
-## 8. Tell us what you’ve tested and checked
+## 8. Test that your changes work in the GOV.UK Design System
```

Comment:
We should probably mark this as an optional step - not all changes will need to be tested against the GOV.UK Design System

Reply:
Sounds good! Will add in.